### ISSUE TITLE
Need Some help with this

### ISSUE THREAD

**Issue by bopaloni** [MultiMC/MultiMC5#1676]
System Information
-----------------------------
MultiMC version: 0.4.11

Operating System: Windows 8.1

Summary of the issue or suggestion: Exit Code -1 Not Starting
----------------------------------------------

What should happen:
------------------------------
Start The Game With Not Enough items installed

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1.  Make a 1.10.2 Instance and use Forge ver 1.10.2-12.181.2095

2. Add N.E.I 1.10.2-2.1.1.148-universal and CodeChickenCore 1.10.2-2.1.8.79-universal 

3.Attempt to start game

Suspected cause: I don't know
---------------------------

Logs/Screenshots: http://paste.ee/p/4Xe1T#
----------------------------

Additional Info: I have been using Multi Mc for a while but this is the first time i don't know what to do
---------------------------

**Comment by Eufranio** [comment]
Mod NotEnoughItems (Not Enough Items) requires [JEI@[3.12.1.287,)]

**Comment by peterix** [comment]
Uh, what?

So NEI now requires JEI?

Just use JEI then.

**Comment by Eufranio** [comment]
Yes. Actually, NEI = tweaks and utilities, JEI = recipes. I like both.
